### DISCUSSION TITLE
[预览] v2.5.2

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3824] [分享 / Share]
包含 v2.5.2 的所有更新内容.

- 优化 `简化评论区` 的时间样式. (#3541, PR #3791 by [Tinhone](https://github.com/Tinhone))
- `下载视频` 支持选择备用下载地址. (PR #3798 by [Seaward233](https://github.com/Seaward233))
- 优化 `自定义顶栏` 的弹窗逻辑, *有任意元素聚焦时* -> *仅在输入框聚焦时*显示弹窗. (#3756)
- `删除广告` 支持屏蔽动态的商品推荐. (#3778)
- 新增插件 `快捷键扩展 - 开关弹幕列表`, 默认为 <kbd>Shift</kbd> + <kbd>D</kbd>. (#3613)
- `展开弹幕列表` 新增 `有选集时不自动展开` 选项. (#1734)
- 新增功能 `禁止滚轮调音量`. (#3489)
> 在网页全屏 / 全屏模式下, 禁止鼠标滚轮控制播放器的音量.

- `下载视频` 新增 `StreamSaver` 输出方式.
- `下载视频` 的 `flv` 格式恢复 1080P 支持. (一共就 360P / 1080P, 其他清晰度还是没有, 需要的话请使用 dash 格式)
- Comment API 支持新版播放页的评论区, `复制评论链接` / `简化评论区` 等相关功能应该可以使用了. (#3807)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.5.2-preview'>[预览] v2.5.2</a>.</em>

**Comment by GalaxySnail** [comment]
似乎又遇到了和[之前](https://github.com/the1812/Bilibili-Evolved/discussions/3461#discussioncomment-3037970)一样的问题，“关于”页面的版本号仍然是旧版：

![](https://user-images.githubusercontent.com/53637421/202856714-b9524bc5-1ea9-4e4a-bad4-e524c8c6d676.png)

**Reply by the1812** [reply]
再试试, 我又更新了一下

**Reply by GalaxySnail** [reply]
OK，解决了

**Comment by f59375543** [comment]
请问历史记录无法显示专栏的问题找到原因了吗？

**Reply by the1812** [reply]
原因知道，只是还没修

**Comment by Demonwls** [comment]
下载视频只能m4s格式，没有声音

**Comment by QRQF001** [comment]
下载FLV格式比较省事，音频视频在一起不用手动合并，但现在flv只有1080P了，没有更高的清晰度就很难受

**Comment by BingLianss** [comment]
无法下载视频，弹出窗口点进去没有下载，新标签下载也不好使，是处bug了吗？

**Comment by BingLianss** [comment]
下载视频弹出的窗口点进去只能播放视频，没有下载是咋回事求回复

**Reply by Geeyun-JY3** [reply]
善用搜索，有相关 issue：#3765

不用发两次，能看到的。

**Comment by Seaward233** [comment]
使用StreamSaver下载视频时，如果用户在浏览器设置中阻止了第三方Cookie，StreamSaver的ServiceWorker将无法成功注册，视频将无法下载，过程中除控制台输出以外没有其他提示。
```
Uncaught (in promise) DOMException: Failed to get a ServiceWorkerRegistration: The user denied permission to use Service Worker.
```
添加例外后恢复正常
![image](https://user-images.githubusercontent.com/35931149/206688222-74a5f6eb-3a11-4fdb-aa05-e8a976a8db0d.png)
